src/components/UI/Dropdown/Dropdown.tsx

```diff
@@ -6,27 +6,28 @@ export interface DropdownProps {
   children: React.ReactNode;
   className?: string;
   alignRight?: boolean;
-  fullWidth?: boolean;
+  shared?: string;
```

Comment:
can maybe rename it to sharedCSS or sharedStyles, if not in future will forget whats this shared thing.